Makefile

```diff
@@ -93,7 +93,7 @@ TEST_INFRA_PROJECT ?= kpt-config-sync-ci-artifacts
 TEST_INFRA_REGISTRY ?= $(LOCATION)-docker.pkg.dev/$(TEST_INFRA_PROJECT)/test-infra
 
 # Docker image used for build and test. This image does not support CGO.
-BUILDENV_IMAGE ?= $(TEST_INFRA_REGISTRY)/buildenv:v0.2.12
```

Comment:
It looks like this image needs to be manually published before the PR is submitted (since it's used by presubmits), which is a bit of an anti-pattern. I can revisit this another time and try to improve this

Reply:
Yup, I have manually pushed the image, but still seeing presubmit failures. Looking now